pages/dkp/kommander/2.1/security/generate-token/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: layout.pug
+navigationTitle: Generate a Client Access Token
+title: Generate a Client Access Token
+menuWeight: 40
+excerpt: How to generate a client access token
+beta: false
+enterprise: false
+---
+
+Obtain a token from the Kommander dashboard, on the user profile dropdown menu.
+
+![Kommander-dashboard](./kommander-generate-token.png)
```

Comment:
Maybe we should put this image after step 1.  Select the **Generate Token** button?